Game/src/main.cpp

```diff
@@ -160,8 +160,6 @@ int main(int argc, char* argv[]) {
 	ret = Run(argc, wargv);
 #endif
 
-	ret = Run(argc, wargv);
-
```

Comment:
@Estrol I'm sorry, I removed the wrong one! I should've removed the `#else` branch above instead of this. 
Somehow I didn't notice that the branch under _WIN32 is commented out, so it looks like nothing is going to happen on game start on Windows.